internal/util/http.go

```diff
@@ -36,7 +36,9 @@ type HttpClient struct {
 func NewHttpClient(logger *log.Logger) HttpClient {
 	return HttpClient{
 		client: &http.Client{
-			Timeout: 10 * time.Second,
+			Transport: &http.Transport{
```

Comment:
Fixed. I though empty value for Transport would be the same as DefaultTransport, but it turns out its not. Copied the default values, expect the ones which don't apply or aren't in go 1.5.